req_fixture.go

```diff
@@ -4,13 +4,13 @@ import "fmt"
 
 type FixtureRequest struct{ request }
 
-// Modifies the request to specify the number of former games to be analyzed (normally 10).
+// Head2Head Modifies the request to specify the number of former games to be analyzed (normally 10).
```

Comment:
I guess "modifies" should be written lowercase here - other than that I think I will apply this generally for all methods with the next commits.